src/formatters.ts

```diff
@@ -9,39 +9,102 @@ type FormatterFunc = (tool: Tool, root: string, document: vscode.TextDocument) =
 type FormatterDefinition = {
   format: FormatterFunc;
   check: ToolCheckFunc;
+  priority: number;
 };
 
 const formatters: Record<FormattingProvider, FormatterDefinition> = {
   muon: {
     format: muon.format,
     check: muon.check,
+    priority: 1, // higher means less priority
```

Comment:
Hmm, I would expect a higher priority value to mean higher priority, similar to `z-index`